launcher/DesktopServices.cpp

```diff
@@ -37,7 +37,7 @@
 #include <QDesktopServices>
 #include <QProcess>
 #include <QDebug>
-#include "Application.h"
+//#include "Application.h"
```

Comment:
?

Reply:
sorry, this header isn't needed here was was causing link problems with the filelink exe. I commented it out to be sure nothing else broke before removing it but forgot about it.